### ISSUE TITLE
v0.0.0.72 GUI SIGSEGV on a regular send

### ISSUE THREAD

**Issue by walkjivefly** [Crowndev/crown#203]
Tried to do a regular send in the QT wallet on linux, it crashed with SIGSEGV.
tail of the debug.log says
```
ernal)
2022-11-16T20:45:15Z [simple] keypool reserve 19
2022-11-16T20:45:15Z [simple] keypool keep 19
2022-11-16T20:45:15Z [simple] Fee Calculation: Fee:295 Bytes:295 Needed:295 Tgt:6 (requested 6) Reason:"Fallback fee" Decay 0.96200: Estimation: (-1 - -1) 0.00% 0.0/(0.0 0 mem 0.0 out) Fail: (0 - 1e+99) 100.00% 1.2/(1.2 0 mem 0.0 out)
2022-11-16T20:45:15Z [simple] keypool added 1 keys (1 internal), size=20 (10 internal)
2022-11-16T20:45:15Z [simple] keypool reserve 20
2022-11-16T20:45:15Z [simple] keypool keep 20
2022-11-16T20:45:15Z [simple] Fee Calculation: Fee:295 Bytes:295 Needed:295 Tgt:6 (requested 6) Reason:"Fallback fee" Decay 0.96200: Estimation: (-1 - -1) 0.00% 0.0/(0.0 0 mem 0.0 out) Fail: (0 - 1e+99) 100.00% 1.2/(1.2 0 mem 0.0 out)
2022-11-16T20:45:15Z [simple] Fee non-grouped = 295, grouped = 295, using grouped
```
gdb says
```
Core was generated by `crown-qt'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x000056102674ff14 in WalletModelTransaction::reassignAmounts(int) ()
[Current thread is 1 (Thread 0x7f0b405cc800 (LWP 1447))]
(gdb) bt
#0  0x000056102674ff14 in WalletModelTransaction::reassignAmounts(int) ()
#1  0x0000561026746d93 in WalletModel::prepareTransaction(WalletModelTransaction&, CCoinControl const&) ()
#2  0x00005610267ca2ba in SendCoinsDialog::PrepareSendText(QString&, QString&, QString&) ()
#3  0x00005610267ce4ce in SendCoinsDialog::on_sendButton_clicked() ()
#4  0x000056102677ca53 in SendCoinsDialog::qt_metacall(QMetaObject::Call, int, void**) ()
#5  0x00005610274a3568 in void doActivate<false>(QObject*, int, void**) ()
#6  0x00005610277e41b5 in QAbstractButtonPrivate::emitClicked() ()
#7  0x00005610277e5f4b in QAbstractButtonPrivate::click() ()
#8  0x00005610277e6167 in QAbstractButton::mouseReleaseEvent(QMouseEvent*) ()
#9  0x00005610277651a8 in QWidget::event(QEvent*) ()
#10 0x000056102772c533 in QApplicationPrivate::notify_helper(QObject*, QEvent*)
    ()
#11 0x0000561027734ed2 in QApplication::notify(QObject*, QEvent*) ()
#12 0x0000561027478288 in QCoreApplication::sendSpontaneousEvent(QObject*, QEvent*) ()
#13 0x00005610277338b6 in QApplicationPrivate::sendMouseEvent(QWidget*, QMouseEvent*, QWidget*, QWidget*, QWidget**, QPointer<QWidget>&, bool, bool) ()
#14 0x000056102777e375 in QWidgetWindow::handleMouseEvent(QMouseEvent*) ()
#15 0x00005610277815b5 in QWidgetWindow::event(QEvent*) ()
#16 0x000056102772c533 in QApplicationPrivate::notify_helper(QObject*, QEvent*)
    ()
#17 0x0000561027478288 in QCoreApplication::sendSpontaneousEvent(QObject*, QEvent*) ()
#18 0x0000561026f356bf in QGuiApplicationPrivate::processMouseEvent(QWindowSystemInterfacePrivate::MouseEvent*) ()
#19 0x00005610271bfa9c in QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#20 0x000056102761ab97 in QXcbUnixEventDispatcher::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#21 0x0000561027476486 in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) ()
#22 0x000056102747f988 in QCoreApplication::exec() ()
#23 0x00005610266c431a in GuiMain(int, char**) ()
#24 0x00007f0b406fcd90 in __libc_start_call_main (
    main=main@entry=0x5610266a5e60 <main>, argc=argc@entry=1, 
    argv=argv@entry=0x7ffc802555f8)
    at ../sysdeps/nptl/libc_start_call_main.h:58
#25 0x00007f0b406fce40 in __libc_start_main_impl (main=0x5610266a5e60 <main>, 
    argc=1, argv=0x7ffc802555f8, init=<optimized out>, fini=<optimized out>, 
    rtld_fini=<optimized out>, stack_end=0x7ffc802555e8)
    at ../csu/libc-start.c:392
#26 0x00005610266bdd25 in _start ()

(gdb) info threads
  Id   Target Id                        Frame 
* 1    Thread 0x7f0b405cc800 (LWP 1447) 0x000056102674ff14 in WalletModelTransaction::reassignAmounts(int) ()
  2    Thread 0x7f0b3f5ed640 (LWP 1448) 0x00007f0b407ebd7f in __GI___poll (
    fds=0x7f0b3f5ecc08, nfds=1, timeout=-1)
    at ../sysdeps/unix/sysv/linux/poll.c:29
  3    Thread 0x7f0b3d5e8640 (LWP 1451) 0x00007f0b407ebe7e in __ppoll (
    fds=0x7f0b28023868, nfds=8, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  4    Thread 0x7f0b3cab7640 (LWP 1452) 0x00007f0b407ebe7e in __ppoll (
    fds=0x7f0af32067e8, nfds=1, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  5    Thread 0x7f0b257fc640 (LWP 1454) __futex_abstimed_wait_common64 (
    private=-1256004864, cancel=true, abstime=0x7f0b257fbc40, op=393, 
    expected=0, futex_word=0x7f0b2c001a58) at ./nptl/futex-internal.c:57
  6    Thread 0x7f0b1ffff640 (LWP 1456) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7f0b2c05de00) at ./nptl/futex-internal.c:57
  7    Thread 0x7f0b1effd640 (LWP 1458) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7f0b2c05de00) at ./nptl/futex-internal.c:57
  8    Thread 0x7f0b0a828640 (LWP 1465) 0x00007f0b407ebd7f in __GI___poll (
    fds=0x7f0b34013890, nfds=11, timeout=50)
    at ../sysdeps/unix/sysv/linux/poll.c:29
  9    Thread 0x7f0b03fff640 (LWP 1470) __futex_abstimed_wait_common64 (
    private=67103320, cancel=true, abstime=0x7f0b03ffec40, op=137, expected=0, 
    futex_word=0x7f0b2ccfe5b4) at ./nptl/futex-internal.c:57
  10   Thread 0x7f0b02ffd640 (LWP 1472) 0x00007f0b407ebe7e in __ppoll (
    fds=0x7f0af40014f8, nfds=1, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  11   Thread 0x7f0b25ffd640 (LWP 1453) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x56102895da90 <scriptcheckqueue+80>)
    at ./nptl/futex-internal.c:57
  12   Thread 0x7f0b24ffb640 (LWP 1455) 0x00007f0b407f8fde in epoll_wait (
    epfd=20, events=0x7f0b2c0709b0, maxevents=32, timeout=2100000)
    at ../sysdeps/unix/sysv/linux/epoll_wait.c:30
  13   Thread 0x7f0b1f7fe640 (LWP 1457) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7f0b2c05de00) at ./nptl/futex-internal.c:57
  14   Thread 0x7f0b1e7fc640 (LWP 1459) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7f0b2c05de00) at ./nptl/futex-internal.c:57
  15   Thread 0x7f0b3dde9640 (LWP 1464) 0x00007f0b407f8fde in epoll_wait (
    epfd=49, events=0x7f0af31ff010, maxevents=32, timeout=656840)
    at ../sysdeps/unix/sysv/linux/epoll_wait.c:30
  16   Thread 0x7f0b09826640 (LWP 1467) __futex_abstimed_wait_common64 (
    private=-385471656, cancel=true, abstime=0x7f0b09825b30, op=137, 
    expected=0, futex_word=0x7f0b2ccfe610) at ./nptl/futex-internal.c:57
--Type <RET> for more, q to quit, c to continue without paging--
  17   Thread 0x7f0b027fc640 (LWP 1474) 0x00007f0b407ebe7e in __ppoll (
    fds=0x7f0b04001068, nfds=1, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  18   Thread 0x7f0b09025640 (LWP 1468) __futex_abstimed_wait_common64 (
    private=680918488, cancel=true, abstime=0x7f0b09024b00, op=137, 
    expected=0, futex_word=0x7f0b2ccfe610) at ./nptl/futex-internal.c:57
  19   Thread 0x7f0b037fe640 (LWP 1471) 0x00007f0b407ebe7e in __ppoll (
    fds=0x7f0ae4001438, nfds=1, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  20   Thread 0x7f0b08824640 (LWP 1469) __futex_abstimed_wait_common64 (
    private=1797285236, cancel=true, abstime=0x7f0b088234f0, op=137, 
    expected=0, futex_word=0x7f0b2ccfe610) at ./nptl/futex-internal.c:57
(gdb) Quit
```

**Comment by walkjivefly** [comment]
Possibly relevant: this was not using the default wallet. crown-qt was started with the default wallet, I used the menu options to unload it, then reload the wallet to use for asset creation. Then created a new contract. Then tried to use send to return the contract funds from change address back to the contract address. This was the send which crashed.

debug.log and coredump [here](https://storage.crownplatform.com/s/HtfEoXw69k5bMPc)

**Comment by BlockMechanic** [comment]
Found and resolved , thanks

**Comment by walkjivefly** [comment]
Nope, still happening in 0.0.0.74
```
Core was generated by `crown-qt'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x000055b696f47b96 in WalletModel::getAddressTableModel() ()
[Current thread is 1 (Thread 0x7fd03492b800 (LWP 19655))]
(gdb) bt
#0  0x000055b696f47b96 in WalletModel::getAddressTableModel() ()
#1  0x000055b696fe9573 in CreateNodeDialog::accept() ()
#2  0x000055b697ca5ea5 in void doActivate<false>(QObject*, int, void**) ()
#3  0x000055b69805fb42 in QDialogButtonBoxPrivate::_q_handleButtonClicked() ()
#4  0x000055b697ca5ea5 in void doActivate<false>(QObject*, int, void**) ()
#5  0x000055b697fe67d5 in QAbstractButtonPrivate::emitClicked() ()
#6  0x000055b697fe856b in QAbstractButtonPrivate::click() ()
#7  0x000055b697fe8787 in QAbstractButton::mouseReleaseEvent(QMouseEvent*) ()
#8  0x000055b697f677c8 in QWidget::event(QEvent*) ()
#9  0x000055b697f2eb53 in QApplicationPrivate::notify_helper(QObject*, QEvent*)
    ()
#10 0x000055b697f374f2 in QApplication::notify(QObject*, QEvent*) ()
#11 0x000055b696f796bd in CrownApplication::notify(QObject*, QEvent*) ()
#12 0x000055b697c7a8a8 in QCoreApplication::sendSpontaneousEvent(QObject*, QEvent*) ()
#13 0x000055b697f35ed6 in QApplicationPrivate::sendMouseEvent(QWidget*, QMouseEvent*, QWidget*, QWidget*, QWidget**, QPointer<QWidget>&, bool, bool) ()
#14 0x000055b697f80995 in QWidgetWindow::handleMouseEvent(QMouseEvent*) ()
#15 0x000055b697f83bd5 in QWidgetWindow::event(QEvent*) ()
#16 0x000055b697f2eb53 in QApplicationPrivate::notify_helper(QObject*, QEvent*)
    ()
#17 0x000055b696f796bd in CrownApplication::notify(QObject*, QEvent*) ()
#18 0x000055b697c7a8a8 in QCoreApplication::sendSpontaneousEvent(QObject*, QEvent*) ()
#19 0x000055b697737cdf in QGuiApplicationPrivate::processMouseEvent(QWindowSystemInterfacePrivate::MouseEvent*) ()
#20 0x000055b6979c20bc in QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#21 0x000055b697e1d1b7 in QXcbUnixEventDispatcher::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#22 0x000055b697c78aa6 in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) ()
#23 0x000055b6980c0b92 in QDialog::exec() ()
#24 0x000055b696f57e08 in WalletView::checkAndCreateNode(COutput const&) ()
#25 0x000055b696f59940 in WalletView::processNewTransaction(QModelIndex const&, int, int) ()
#26 0x000055b697ca5e71 in void doActivate<false>(QObject*, int, void**) ()
#27 0x000055b697c2f467 in QAbstractItemModel::endInsertRows() ()
#28 0x000055b696ff3c4f in TransactionTablePriv::updateWallet(interfaces::Wallet&, uint256 const&, int, bool) ()
#29 0x000055b696ff20a6 in TransactionTableModel::updateTransaction(QString const&, int, bool) ()
#30 0x000055b697ca171e in QObject::event(QEvent*) ()
#31 0x000055b697f2eb53 in QApplicationPrivate::notify_helper(QObject*, QEvent*)
    ()
#32 0x000055b696f796bd in CrownApplication::notify(QObject*, QEvent*) ()
#33 0x000055b697c79f8a in QCoreApplication::notifyInternal2(QObject*, QEvent*)
    ()
#34 0x000055b697c7df0b in QCoreApplicationPrivate::sendPostedEvents(QObject*, in--Type <RET> for more, q to quit, c to continue without paging--
t, QThreadData*) ()
#35 0x000055b697cc6ba5 in QEventDispatcherUNIX::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#36 0x000055b697e1d1a2 in QXcbUnixEventDispatcher::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#37 0x000055b696fcbae4 in SendCoinsDialog::addEntry() ()
#38 0x000055b696fcbf87 in SendCoinsDialog::clear() ()
#39 0x000055b696fd25cc in SendCoinsDialog::on_sendButton_clicked() ()
#40 0x000055b696f81063 in SendCoinsDialog::qt_metacall(QMetaObject::Call, int, void**) ()
#41 0x000055b697ca5b88 in void doActivate<false>(QObject*, int, void**) ()
#42 0x000055b697fe67d5 in QAbstractButtonPrivate::emitClicked() ()
#43 0x000055b697fe856b in QAbstractButtonPrivate::click() ()
#44 0x000055b697fe8787 in QAbstractButton::mouseReleaseEvent(QMouseEvent*) ()
#45 0x000055b697f677c8 in QWidget::event(QEvent*) ()
#46 0x000055b697f2eb53 in QApplicationPrivate::notify_helper(QObject*, QEvent*)
    ()
#47 0x000055b697f374f2 in QApplication::notify(QObject*, QEvent*) ()
#48 0x000055b696f796bd in CrownApplication::notify(QObject*, QEvent*) ()
#49 0x000055b697c7a8a8 in QCoreApplication::sendSpontaneousEvent(QObject*, QEvent*) ()
#50 0x000055b697f35ed6 in QApplicationPrivate::sendMouseEvent(QWidget*, QMouseEvent*, QWidget*, QWidget*, QWidget**, QPointer<QWidget>&, bool, bool) ()
#51 0x000055b697f80995 in QWidgetWindow::handleMouseEvent(QMouseEvent*) ()
#52 0x000055b697f83bd5 in QWidgetWindow::event(QEvent*) ()
#53 0x000055b697f2eb53 in QApplicationPrivate::notify_helper(QObject*, QEvent*)
    ()
#54 0x000055b696f796bd in CrownApplication::notify(QObject*, QEvent*) ()
#55 0x000055b697c7a8a8 in QCoreApplication::sendSpontaneousEvent(QObject*, QEvent*) ()
#56 0x000055b697737cdf in QGuiApplicationPrivate::processMouseEvent(QWindowSystemInterfacePrivate::MouseEvent*) ()
#57 0x000055b6979c20bc in QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#58 0x000055b697e1d1b7 in QXcbUnixEventDispatcher::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) ()
#59 0x000055b697c78aa6 in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) ()
#60 0x000055b697c81fa8 in QCoreApplication::exec() ()
#61 0x000055b696ec83ea in GuiMain(int, char**) ()
#62 0x00007fd034a5bd90 in __libc_start_call_main (
    main=main@entry=0x55b696ea9f30 <main>, argc=argc@entry=1, 
    argv=argv@entry=0x7ffea8798268)
    at ../sysdeps/nptl/libc_start_call_main.h:58
#63 0x00007fd034a5be40 in __libc_start_main_impl (main=0x55b696ea9f30 <main>, 
    argc=1, argv=0x7ffea8798268, init=<optimized out>, fini=<optimized out>, 
    rtld_fini=<optimized out>, stack_end=0x7ffea8798258)
    at ../csu/libc-start.c:392
#64 0x000055b696ec1df5 in _start ()
(gdb) info threads
  Id   Target Id                         Frame 
* 1    Thread 0x7fd03492b800 (LWP 19655) 0x000055b696f47b96 in WalletModel::getAddressTableModel() ()
  2    Thread 0x7fd030e16640 (LWP 19660) 0x00007fd034b4ae7e in __ppoll (
    fds=0x7fd0219f5d78, nfds=1, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  3    Thread 0x7fd019ffd640 (LWP 19661) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x55b69915fa94 <scriptcheckqueue+84>)
    at ./nptl/futex-internal.c:57
  4    Thread 0x7fd031947640 (LWP 19659) 0x00007fd034b4ae7e in __ppoll (
    fds=0x7fd01c017a78, nfds=6, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  5    Thread 0x7fd0197fc640 (LWP 19662) __futex_abstimed_wait_common64 (
    private=-974323200, cancel=true, abstime=0x7fd0197fbc40, op=393, 
    expected=0, futex_word=0x7fd02000157c) at ./nptl/futex-internal.c:57
  6    Thread 0x7fd018ffb640 (LWP 19663) 0x00007fd034b57fde in epoll_wait (
    epfd=20, events=0x7fd02006ff10, maxevents=32, timeout=-1)
    at ../sysdeps/unix/sysv/linux/epoll_wait.c:30
  7    Thread 0x7fd013fff640 (LWP 19664) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7fd02004d5f0) at ./nptl/futex-internal.c:57
  8    Thread 0x7fd011ffb640 (LWP 19668) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x55b699169838 <leveldb::Env::Default()::env_container+88>)
    at ./nptl/futex-internal.c:57
  9    Thread 0x7fd0137fe640 (LWP 19665) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7fd02004d5f0) at ./nptl/futex-internal.c:57
  10   Thread 0x7fd012ffd640 (LWP 19666) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7fd02004d5f0) at ./nptl/futex-internal.c:57
  11   Thread 0x7fcffffff640 (LWP 19671) 0x00007fd034b57fde in epoll_wait (
    epfd=50, events=0x7fd0219ee620, maxevents=32, timeout=291929)
    at ../sysdeps/unix/sysv/linux/epoll_wait.c:30
  12   Thread 0x7fd0127fc640 (LWP 19667) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7fd02004d5f4) at ./nptl/futex-internal.c:57
  13   Thread 0x7fcffe7fc640 (LWP 19674) __futex_abstimed_wait_common64 (
    private=0, cancel=true, abstime=0x0, op=393, expected=0, 
    futex_word=0x7fd0219f0088) at ./nptl/futex-internal.c:57
  14   Thread 0x7fcffdffb640 (LWP 19675) __futex_abstimed_wait_common64 (
    private=-1726603816, cancel=true, abstime=0x7fcffdffab00, op=137, 
    expected=0, futex_word=0x7fd0201606f0) at ./nptl/futex-internal.c:57
  15   Thread 0x7fcfe7fff640 (LWP 19678) 0x00007fd034b4ae7e in __ppoll (
    fds=0x7fcff8001118, nfds=1, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  16   Thread 0x7fcfe77fe640 (LWP 19679) 0x00007fd034b4ae7e in __ppoll (
    fds=0x7fd004001038, nfds=1, timeout=<optimized out>, sigmask=0x0)
--Type <RET> for more, q to quit, c to continue without paging--
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  17   Thread 0x7fcfe6ffd640 (LWP 19681) 0x00007fd034b4ae7e in __ppoll (
    fds=0x7fd000001068, nfds=1, timeout=<optimized out>, sigmask=0x0)
    at ../sysdeps/unix/sysv/linux/ppoll.c:42
  18   Thread 0x7fcffcff9640 (LWP 19677) __futex_abstimed_wait_common64 (
    private=-50361768, cancel=true, abstime=0x7fcffcff8c40, op=137, 
    expected=0, futex_word=0x7fd020160690) at ./nptl/futex-internal.c:57
  19   Thread 0x7fd03394c640 (LWP 19656) 0x00007fd034b4ad7f in __GI___poll (
    fds=0x7fd03394bc08, nfds=1, timeout=-1)
    at ../sysdeps/unix/sysv/linux/poll.c:29
  20   Thread 0x7fcfff7fe640 (LWP 19672) 0x00007fd034b4ad7f in __GI___poll (
    fds=0x55b69b959550, nfds=11, timeout=50)
    at ../sysdeps/unix/sysv/linux/poll.c:29
  21   Thread 0x7fcffd7fa640 (LWP 19676) __futex_abstimed_wait_common64 (
    private=1797285236, cancel=true, abstime=0x7fcffd7f94f0, op=137, 
    expected=0, futex_word=0x7fd0201606f0) at ./nptl/futex-internal.c:57
```

**Comment by walkjivefly** [comment]
although, tbf it was a 10k transfer and it did ask if I wanted to create a MN so maybe it was actually #187 which is also still happening.

**Comment by BlockMechanic** [comment]
I have disabled 10K and 500 ->MN/SN creation detection in regular send. Should eliminate that variable. I was getting too fancy.

**Comment by walkjivefly** [comment]
Confirmed fixed in 0.0.0.75.
Thank you.
